third-party/aws-sdk-cpp/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2014-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed in accordance with the terms specified in
+# the LICENSE file found in the root directory of this source tree.
+
+cmake_minimum_required(VERSION 3.13.3)
+
+function(awsSdkCppMain)
+  set(name "aws-sdk-cpp")
+  set(version "1.4.55")
+
+  set(anchor_file "lib/libtesting-resources.a")
+
+  set(additional_libraries
```

Comment:
Any thoughts about how to make this more generic? 

For example you are copy-pasting the library paths save for the {a,lib}.

Reply:
Yeah, in general this can be simplified even more.
If we are sure that all the .a or .lib present in the pack are needed, we could just GLOB them and use the same "importThirdPartyBinaryLibrary" to generate all the needed targets.